apps/frontend/src/components/observations/plant-observation-form.tsx

```diff
@@ -0,0 +1,227 @@
+import { Button } from "@/components/ui/button";
+import DatePicker from "@/components/ui/datepicker";
+import { Dialog, DialogContent, DialogHeader, DialogTitle, DialogTrigger } from "@/components/ui/dialog";
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { Select, SelectContent, SelectItem, SelectTrigger } from "@/components/ui/select";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import { PlantInformation } from "types/observations";
+
+export default function PlantObservationForm(
+  { newPlant }: { newPlant: boolean },
+) {
+  const [open, setopen] = useState(false);
+
+  const form = useForm<PlantInformation>({
+    defaultValues: {
+      commonName: '',
+      scientificName: null,
+      plantType: null,
+      native: false,
+      quantity: null,
+      soilType: null,
+      dateBloomed: null,
+      datePlanted: null
+    }
+  });
+  
+  const onSubmit = (data: PlantInformation) => {
+    console.log("Submitted", data);
+    setopen(false);
+  };
+
+  return (
+    <Dialog open={open} onOpenChange={setopen}>
+      <DialogTrigger asChild>
+        <Button variant="outline">
+          {newPlant ? "New Plant" : "Editing Plant"}
+        </Button>
+      </DialogTrigger>
+
```

Comment:
Noticing a bunch of the CSS looks off, but we can fix in later PR because of the showcase